tests/test_management_commands.py

```diff
@@ -0,0 +1,481 @@
+
+        # Check the command output
+        self.assertIn("Looking for image assets modified in the last 3 hour(s)", output)
+
+        # Check the mocked API was called as expected
+        self.mock_api_client.asset_bank_client.media_list.assert_called_once_with(
+            {
+                "dateModified": expected_datemodified.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                **self.common_api_kwargs,
+            }
+        )
+        self.mock_api_client.asset_bank_client.media_info.assert_called_once_with(
+            TEST_ASSET_ID
+        )
+
+        # Check the patched object was updated and saved as expected
+        self.patched_obj.update_from_asset_data.assert_called_once_with(TEST_ASSET_DATA)
+        self.patched_obj.save.assert_called_once()
+
+    def test_days(self):
+        expected_datemodified = datetime.datetime.utcnow() - datetime.timedelta(days=3)
+        output = self.call_command(days=3)
+
+        # Check the command output
+        self.assertIn("Looking for image assets modified in the last 3 day(s)", output)
+
+        # Check the mocked API was called as expected
+        self.mock_api_client.asset_bank_client.media_list.assert_called_once_with(
+            {
+                "dateModified": expected_datemodified.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                **self.common_api_kwargs,
+            }
+        )
+        self.mock_api_client.asset_bank_client.media_info.assert_called_once_with(
+            TEST_ASSET_ID
+        )
+
+        # Check the patched object was updated and saved as expected
+        self.patched_obj.update_from_asset_data.assert_called_once_with(TEST_ASSET_DATA)
+        self.patched_obj.save.assert_called_once()
+
+
+@freeze_time()
+class UpdateStaleDocumentsTestCase(BaseSyncCommandTestCase):
+    """
+    Unit tests for the 'update_stale_documents' management command.
+    """
+
+    command_name = "update_stale_documents"
+    model_class = get_document_model()
+    common_api_kwargs = {
+        "orderBy": "dateModified desc",
+        "page": 1,
+        "limit": 200,
+        "type": "document",
+    }
+
+    def test_default(self):
+        expected_datemodified = datetime.datetime.utcnow() - datetime.timedelta(days=1)
+        output = self.call_command()
+
+        # Check the command output
+        self.assertIn(
+            "Looking for document assets modified in the last 1 day(s)", output
+        )
+
+        # Check the mocked API was called as expected
+        self.mock_api_client.asset_bank_client.media_list.assert_called_once_with(
+            {
+                "dateModified": expected_datemodified.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                **self.common_api_kwargs,
+            }
+        )
+
+        # Check the patched object was updated and saved as expected
+        self.patched_obj.update_from_asset_data.assert_called_once_with(TEST_ASSET_DATA)
+        self.patched_obj.save.assert_called_once()
+
+    def test_minutes(self):
```

Comment:
it may be nice to test the precedence. so if I call any of the commands with 

- `--minutes 4 --hours 3 --days 2` -> days is considered
- `--minutes 4 --hours 3` -> hours is considered